app/templates/main/search-transferring-body.html

```diff
@@ -90,36 +91,62 @@ <h2 class="govuk-body browse__body">You are viewing</h2>
                                                 {% for record in results %}
                                                     {% set main_loop_index = loop.index %}
                                                     <tr class="govuk-table__row govuk-table__row--primary">
+                                                        {% set first_result_key = record['highlight'].keys() | list | first | replace('_', ' ') | capitalize %}
+                                                        {% set first_result_value = record['highlight'].values() | list | first | join(' ... ')| clean_tags | safe %}
+                                                        {% set field_count = record['highlight'].keys() | length - 1 %}
                                                         <td class="govuk-table__cell">
                                                             <div class="govuk-table__cell__data">
                                                                 <details class="details-toggle-{{ main_loop_index }}"
                                                                          {% if open_all %}open{% endif %}>
                                                                     <summary />
                                                                 </details>
                                                                 {% if record['highlight'] %}
-                                                                    {% set key = record['highlight'].keys() | list | first | replace('_', ' ') | capitalize %}
-                                                                    {{ key }} +{{ record['highlight'].keys() | length - 1 }}
+                                                                    {% if field_count > 0 %}
+                                                                        {{ first_result_key }} +{{ field_count }}
```

Comment:
for consistency could we rename

`first_result_key` -> `first_field`


and change the logic so that filed_count doesnt include the minus 1, and then 
`{% if field_count > 1 %}
and then here you do 
`{{ first_field }} + {{ field_count - 1 }}`